app/templates/power-ups/client-library.html

```diff
@@ -81,48 +62,41 @@ <h3 id="ux-methods">UX Methods</h3>
-	Using `t.set` will re-initialize your Power-Up in it’s entirety by re-executing all capabilities.
-  Read more about data storage in the <a ui-sref="power-ups.topics({'#':'data-storage'})">topics section</a>.
-  </li>
-	<li><strong>get(scope, visibility, name, defaultValue)</strong> Used to get all of the stored data for the Power-Up</li>
-	<li><strong>attach({url:URL, name:Name})</strong> Attach a new URL to the card in the current context.</li>
-  <li><strong>signUrl(URL)</strong> Sign a URL for use with <strong>attachment-sections</strong> only.</li>
-  <li><strong>localizeKey(key, data)</strong> Localize a string by key</li>
-  <li><strong>localizeKeys(keys)</strong> Localize a list of strings or objects. You can read more about internationalization and localization in the <a ui-sref="power-ups.topics({'#':'internationalization'})">topics section</a>.</li>
-  <li><strong>board(field1, field2, ...)</strong>
-	Returns a promise with information about the board for the current context.
-	Valid fields include: 'id', 'name', 'url', 'shortLink', 'members'</li>
-	<li><strong>list(field1, field2, ...)</strong>
-	Returns a promise with information about the list for the current context
-	Valid fields include: 'id', 'name', 'cards'</li>
-	<li><strong>card(field1, field2, ...)</strong>
-	Returns a promise with information about the card for the current context. The information available is only when a specific card is in context. For instance, this function will return information about a card successfully inside of the capabilities 'attachment-sections' and 'card-badges', but not 'show-settings' nor 'board-buttons'. The latter are not within the context of a single card.
-	Valid fields include: 'id', 'name', 'desc', 'due', 'closed', 'cover', 'attachments', 'members', 'labels', 'url', 'shortLink', 'idList', 'idShort'<br /><br />
-	Example usage:
-<pre>t.card('id', 'name', 'url').then(function(promiseResult){console.log(promiseResult)});
+    <li><strong>iframe()</strong> Get access to the t object within an iframe</li>
+    <li><strong>set(scope, visibility, name, value)</strong> Used to store some persistent data in Trello's database. The Power-Up provides a scope (organization, board, or card) and the visibility of the data (i.e. whether it's ‘shared’ with everyone that can see the org/board/card or if it's ‘private’) and a string to store. Power-Ups are allowed to store 1K of data for each scope/visibility pair and the current Power-Ups store stringified JSON. Using `t.set` will re-initialize your Power-Up in it’s entirety by re-executing all capabilities. Read more about data storage in the <a ui-sref="power-ups.topics({'#':'data-storage'})">topics section</a>.
```

Comment:
Much better than having to do a whole bunch of `t.set(...)` calls 1 key at a time.